resources/js/components/Listing/ListingView.vue

```diff
@@ -0,0 +1,124 @@
+        </div>
+
+        <data-list
+            v-if="!initializing"
+            class="mb-4"
+            :visible-columns="columns"
+            :columns="columns"
+            :rows="items"
+            :sort="false"
+            :sort-column="sortColumn"
+            :sort-direction="sortDirection"
+        >
+            <div slot-scope="{ hasSelections }">
+                <div class="card p-0 relative">
+                    <data-list-filter-presets
+                        ref="presets"
+                        :active-preset="activePreset"
+                        :preferences-prefix="preferencesPrefix"
+                        @selected="selectPreset"
+                        @reset="filtersReset"
+                    />
+
+                    <div class="data-list-header">
+                        <data-list-filters
+                            :filters="filters"
+                            :active-preset="activePreset"
+                            :active-preset-payload="activePresetPayload"
+                            :active-filters="activeFilters"
+                            :active-filter-badges="activeFilterBadges"
+                            :active-count="activeFilterCount"
+                            :search-query="searchQuery"
+                            :saves-presets="true"
+                            :preferences-prefix="preferencesPrefix"
+                            @filter-changed="filterChanged"
+                            @search-changed="searchChanged"
+                            @saved="$refs.presets.setPreset($event)"
+                            @deleted="$refs.presets.refreshPresets()"
+                            @restore-preset="$refs.presets.viewPreset($event)"
+                            @reset="filtersReset"
+                        />
+                    </div>
+
+                    <div
+                        v-show="items.length === 0"
+                        class="p-3 text-center text-grey-50"
+                        v-text="__('No results')"
+                    />
+
+                    <data-list-table
+                        v-show="items.length"
+                        :allow-bulk-actions="false"
+                        :loading="loading"
+                        :reorderable="false"
+                        :sortable="true"
+                        :toggle-selection-on-row-click="true"
+                        :allow-column-picker="true"
+                        :column-preferences-key="preferencesKey('columns')"
+                        @sorted="sorted"
+                    >
+                        <template slot="actions" slot-scope="{ row, index }">
+                            <dropdown-list>
+                                <dropdown-item
+                                    :text="__('Edit')"
+                                    :redirect="row.editUrl"
+                                />
+
+                                <dropdown-item
```

Comment:
Not sure why but I'm unable to see the 'Delete' dropdown here.. 

![image](https://user-images.githubusercontent.com/19637309/124079521-2bf2e480-da41-11eb-92fe-358048b2b9bd.png)

Reply:
Should be there in the next push.

Reply:
Whoop whoop, the button shows. Although, there's a really weird behaviour when I try to use it. Loom: https://www.loom.com/share/727e507d35e04a5abed164f292358fd1